modules/reactor/src/meshgenerators/PatternedHexMeshGenerator.C

```diff
@@ -154,10 +155,14 @@ PatternedHexMeshGenerator::PatternedHexMeshGenerator(const InputParameters & par
         "control_drums_azimuthal_meta", std::vector<std::vector<Real>>())),
     _position_file_name(declareMeshProperty<std::string>("position_file_name",
                                                          getParam<std::string>("position_file"))),
+    _hexagon_peripheral_trimmability(
+        declareMeshProperty<bool>("hexagon_peripheral_trimmability", !_generate_core_metadata)),
```

Comment:
so the issue with declaring these trimmabilities boolean is that if someone gives you an exodus mesh to trim you cant really do it, it needs to be fresh from the reactor module or loaded from checkpoint (meshProperty is restartable iirc)

maybe there should be an option to override these booleans?

Reply:
we'd need to add the meta data to the loaded mesh right? is there a generator for that?

Reply:
Currently, it only works for some MGs in Reactor modules for now.

I do have some plan in mind to have a MG to assign new metadata (especially for FileMG). Hopefully it can be done in FY23.

Reply:
ok that works. Assigning metadata will be nice for working with the FMG indeed